components/aws/common/constants.mjs

```diff
@@ -0,0 +1,58 @@
+import { createObjectFromArray } from "./utils.mjs";
+
+const clients = {
+  ec2: "ec2",
+  lambda: "lambda",
+  ssm: "ssm",
+  iam: "iam",
+  s3: "s3",
+  cloudWatchLogs: "cloudWatchLogs",
+  eventBridge: "eventBridge",
+  sqs: "sqs",
+  sns: "sns",
+  ses: "ses",
+  dynamodb: "dynamodb",
+};
+
+const dynamodb = {
+  keyTypes: createObjectFromArray([
```

Comment:
I was thinking if this way of creating a constant is better than the above (`clients`). It is cleaner imo, but there is code completion with the `clients` constant.

Reply:
Yeah, I agree with you. Addressed